packages/react-instantsearch-hooks-dom/src/ui/Pagination.tsx

```diff
@@ -52,26 +96,34 @@ export function Pagination({
           <PaginationItem
             isDisabled={isFirstPage}
             className="ais-Pagination-item--firstPage"
-            aria-label="First"
+            aria-label={translations.ariaFirst}
             href={createURL(firstPageIndex)}
             onClick={() => onNavigate(firstPageIndex)}
           >
-            ‹‹
+            {translations.first}
           </PaginationItem>
         )}
         {showPrevious && (
           <PaginationItem
             isDisabled={isFirstPage}
             className="ais-Pagination-item--previousPage"
-            aria-label="Previous"
+            aria-label={translations.ariaPrevious}
             href={createURL(previousPageIndex)}
             onClick={() => onNavigate(previousPageIndex)}
           >
-            ‹
+            {translations.previous}
           </PaginationItem>
         )}
         {pages.map((page) => {
-          const label = String(page + 1);
+          const label =
+            typeof translations.page === 'string'
+              ? translations.page
+              : translations.page(page);
```

Comment:
For translations it makes sense to provide the actual 1-based page number  imo (first being `1` and not `0`).

Reply:
Addressed in 91703928.